README.rst

```diff
@@ -31,6 +31,30 @@ Documentation
 
 The full documentation is at https://course-access-groups.readthedocs.org.
 
+Quickstart Instructions
+-----------------------
+
+Install this plugin via ``pip``. Then configure your Ansibe
+``server-vars.yml`` with the following:
```

Comment:
Assuming this is just for Appsembler? I forget if `server-vars.yml` is an Open edX standard thing or just us?

Reply:
It's an ideal Open edX configuration setup that's probably just us following it. It has a couple of mentions in the [Open edX configuration](https://github.com/edx/configuration/search?utf8=%E2%9C%93&q=server-vars&type=) and in the [docs](https://github.com/edx/edx-documentation/blob/233dcb93f8a4fbea4788cefe1793128b6837d737/en_us/install_operations/source/configuration/enable_ccx.rst).

It think it's somewhat safe to mention it as long as we specify that it's an `Ansible` thing.